app/service_invite/rest.py

```diff
@@ -189,12 +185,12 @@ def send_service_invite_request(
             # In a scenario were multiple service managers are listed, and the list contains an
             # invalid service manager, we would rather log the errors and not raise an exception so
             # that notifications can still be sent to the valid service managers
-            message = f"Can’t create notification - {recipient.name} is not part of the {service.name}"
+            message = "not-a-valid-service-manager"
             current_app.logger.error(message)
```

Comment:
Given these are log messages rather than responses, we can keep them information rich so it helps us investigate :)



```suggestion
            current_app.logger.error("Ignoring request-to-join-service email to user %(recipient.id) - they are not part of service %(service.id)", dict(recipient=recipient, service=service))
```